### PR TITLE
[MIG] partner_identification_import: Migration to 14.0

### PR DESCRIPTION
**Pull request by dzungtran89**
Dependencies:
- partner_identification: https://github.com/OCA/partner-contact/pull/1058
- base_business_document_import: https://github.com/OCA/edi/pull/301

### CODE REVIEW

File: partner_identification_import/models/business_document_import.py [hunk 1]
```diff
@@ -44,16 +44,18 @@ def _hook_match_partner(self, partner_dict, chatter_msg, domain, order):
...
                 raise self.user_error_wrap(
                     _(
                         "Odoo couldn't find a partner corresponding to the "
                         "following information extracted from the business document:\n"
-                        "%s"
+                        "{}"
                     )
-                    % _("or\n").join(unmatched)
+                    .format_("or\n")
+                    .join(unmatched)
```
Comment (sebalix) [inline partner_identification_import/models/business_document_import.py]:
```suggestion
                    .format(_("or\n").join(unmatched))
```

File: test-requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+odoo14-addon-base-business-document-import @git+https://github.com/trobz/edi@14.0-mig-base_business_document_import#subdirectory=setup/base_business_document_import
+odoo14-addon-partner-identification @git+https://github.com/phuctranerp/partner-contact@14.0-mig-partner_identification#subdirectory=setup/partner_identification
```
Comment (sebalix) [inline test-requirements.txt]: To remove now that dependencies are merged

**Review by sebalix — CHANGES_REQUESTED** [review]
LG, small change requested

File: partner_identification_import/models/business_document_import.py (hunk 1 above)
Comment (dzungtran89) [reply partner_identification_import/models/business_document_import.py]: ok, thx

**Review by simahawk — APPROVED** [review]
@sebalix fine for you?

**Review by sebalix — APPROVED** [review]

**Comment by simahawk** [comment]
oh I missed 3 commits that should be squashed. @dzungtran89 could you do it? Otherwise I do it.

**Comment by dzungtran89** [comment]
Hi @simahawk , I'll do it

File: partner_identification_import/models/business_document_import.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+                    )
+                )
+            if unmatched:
+                raise self.user_error_wrap(
+                    _(
+                        "Odoo couldn't find a partner corresponding to the "
+                        "following information extracted from the business document:\n"
+                        "{}"
+                    )
+                    .format_("or\n")
```
Comment (simahawk) [inline partner_identification_import/models/business_document_import.py:58]: there's typo here. I'll fix it while fwd porting other changes.

**Pull request merged**